TypeScript.YAML-tmLanguage

```diff
@@ -653,7 +653,8 @@ repository:
 
   comment-line:
     name: comment.line.ts
-    match: (//).*$\n?
+    begin: //
+    end: $
```

Comment:
Can you say $ | line_feed (\n) to accommodate for different OSs. Dot (.) excludes line breaks too. Is there a particular reason why you want to switch to this?